scripts/import-osm

```diff
@@ -54,6 +59,7 @@ ${OSM2PGROUTING} -f /tmp/$COUNTRY.osm -c $(dirname "${BASH_SOURCE}")/mapconfig.x
 echo "Populating ways_nodes table"
 psql -d $POSTGRES_DB -U $POSTGRES_USER -h $POSTGRES_HOST << SQL_SCRIPT
 SELECT populate_ways_nodes();
+SELECT connect_isolated_segments(5000);
```

Comment:
I don't like mixing this with migrations. If we add more countries we would run the function again to address isolated segments from the new data. It's also not truly idempotent: after run, we cannot change the threshold.